adops/mobile-rendering-max-line-item-setup.md

```diff
@@ -43,3 +45,8 @@ Make sure that the `Custom Parameters` field contain expecting targetting keywor
 
 {: .alert.alert-warning :}
 The adapter will render the winning bid only if the bid's targeting keywords contain `all` keywords from the `Custom Parameters` field.
+
```

Comment:
old typo in the line 40 contain --> contains

Reply:
fixed